client/src/pages/TeacherForm.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+// import Template from "/Components/Template";
+
+const TeacherForm = () => {
+    return (
+        <div className="d-flex justify-content-center">
+            {/* <Template /> */}
+            <Form>
+                <h1>Register A Teacher</h1>
+                <p>Please enter the details below to register a teacher</p>
+                    
+                <div className="row">
```

Comment:
Maybe you can use the `Row` component from react-bootstrap

Reply:
Thanks, will do and resend